Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -112,7 +112,7 @@ open class ChatMessageContentView: _View, ThemeProvider {
 
     /// Shows the bubble around message reactions.
     /// Exists if `layout(options: MessageLayoutOptions)` was invoked with the options containing `.reactions`.
-    public private(set) var reactionsBubbleView: ChatReactionsBubbleView?
+    public private(set) var reactionsBubbleView: ChatReactionBubbleBaseView?
```

Comment:
This is a breaking change right? 🤔

Reply:
it should not be `ChatReactionsBubbleView ` is a subclass of `ChatReactionBubbleBaseView`